dymos/transcriptions/solve_ivp/solve_ivp.py

```diff
@@ -532,6 +533,18 @@ def configure_timeseries_outputs(self, phase):
                 if var_type != 'ode':
                     continue
 
+                # Skip the timeseries output if it does not appear to be shaped as a dynamic variable
+                # If the full shape does not start with num_nodes, skip this variable.
+                ode_outputs = {opts['prom_name']: opts for (k, opts) in
```

Comment:
Do you need to add this to runge_kutta too?

Reply:
Since it's deprecated I wasn't going to, but I suppose I can add that in.